src/common.js

```diff
@@ -1,7 +1,7 @@
 import ace from 'ace-builds';
 import 'ace-builds/src-min-noconflict/mode-java.js';
 
-// this is going to be the place where all shared enums and constants.
+// this is going to be the place where all shared enums and constants. (this is the place for all shared enums and constants.)
```

Comment:
Is this comment redundant now?

Reply:
one is ben's version, the one I added is my version (that makes more sense to me because ben's original comment didnt seem complete) I just left it there for now